### DISCUSSION TITLE
netbox import all the configurations from old to new netbox

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#13423] [General]
Hi all.

Is there any way that can we export all the configs (site,racks,,connections..) from old netbox to new netbox ? Its getting difficult for me to fix the erros with the old netbox. hence wanted to know if we can somehow export all the configs to new one.

**Comment by xkilian** [comment]
Someone in the know can correct me.
My understanding is that you need to upgrade your old netbox so that your database schema is compatible with your new installation. Then you just copy the database using the backup/restore instructions.

**Reply by lasag-2511** [reply]
No i am having issues with old netbox during the upgradation, hence i wanted to migrate all the data from old to new one with the latest version. For some reason my old netbox webpage doesnot work anymore

**Reply by xkilian** [reply]
Try and migrate your backup from your old version to another equivalent vanilla old version and re-run the upgrade scripts following the recommended upgrade path, you can't upgrade directly to the latest version if it is an older version of Netbox. Plugins may be a cause of problems as well..

**Comment by mtinberg** [comment]
Read carefully through the release notes from each major version (3.0, 3.1, 3.2, ...) between your current install and the target install as there are often specific upgrade instructions and incompatibilities that need to be addressed, eg. the need to do a stepwise upgrade to 2.11.10 before upgrading to 3.x)

Read carefully through the upgrade instructions and pay special attention to the need to capture a backup of the PostgreSQL database using pg_dump before you start changing anything and attending to any database upgrades (to PostgreSQL 12.x for example) highlighted in the release notes.

If you have a good backup of your data, you can always recover, you can't mess up and lose data, even as you make mistakes while learning the process.  I'm just not sure I can bridge the gulf of understanding of how this works in the time I have to give any better advice.

Good luck!


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by lasag-2511** [reply]
fixed by stopping firewall services .
